src/Parser/RestoDruid/Modules/Features/StatWeights.js

```diff
@@ -0,0 +1,256 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import Module from 'Parser/Core/Module';
+import HIT_TYPES from 'Parser/Core/HIT_TYPES';
+import { formatNumber } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+import HealingValue from 'Parser/Core/Modules/HealingValue';
+import CritEffectBonus from 'Parser/Core/Modules/Helpers/CritEffectBonus';
+
+import { getSpellInfo } from '../../SpellInfo';
+
+const DEBUG = true;
+
+const ARMOR_INT_MULTIPLIER = 1.05; // 5% int bonus from wearing all leather means each new point of int worth 1.05 vs character sheet int
+
+class StatWeights extends Module {
+  static dependencies = {
+    combatants: Combatants,
+    critEffectBonus: CritEffectBonus,
+  };
+
+  concordanceAmount = 0; // TODO remove when stat tracker added
+
+  totalAdjustedHealing = 0; // total healing after excluding 'multiplier' spells like Leech / Velens
+
+  // These are the total healing that would be gained if their respective stats ratings were increased by one.
+  totalOneInt = 0;
+  totalOneCrit = 0;
+  totalOneHasteHpm = 0;
+//  totalOneHasteHpct = 0;
+  totalOneMastery = 0;
+  totalOneVers = 0;
+  totalOneLeech = 0;
+
+  on_initialized() {
+    this.concordanceAmount = 4000 + (this.combatants.selected.traitsBySpellId[SPELLS.CONCORDANCE_OF_THE_LEGIONFALL_TRAIT.id] || 0) * 300; // TODO remove when stat tracker added
```

Comment:
Doesn't this include 1 point too many because counting the first point twice?

Reply:
Yeah you're right, I'll fix it.